### DISCUSSION TITLE
Enterprise Cloud trial - multiple organizations

### DISCUSSION THREAD

**Discussion by nickpan-fioneer** [community/community#23463] [Enterprise]
<p>Is it possible to add/invite an organization to an Enterprise Cloud <strong>trial</strong> account?</p>
<p>I currently have a trial account, but don’t see the “Your enterprises” option under my profile photo, per <a href="https://docs.github.com/en/enterprise-cloud@latest/admin/user-management/managing-organizations-in-your-enterprise/adding-organizations-to-your-enterprise" rel="noopener nofollow ugc">this page</a>.</p>

**Comment by ettaboyle** [comment]
<p>Hi <a class="mention" href="/nickpan-fioneer">@nickpan-fioneer</a> Welcome to the forums! Thanks for including that link to the docs in your post. I reached out to a couple of folks to see if there are any specifics around trial accounts.</p>
<p>For this and FYI for the future: most often the best place to go for answers to account-specific questions is opening a ticket on our <a href="https://support.github.com/contact">Support page</a>. We’re limited on what we can share in a public forum.</p>
<p>Thanks and I will circle back on this thread if I find out more!</p>

**Comment by ettaboyle** [comment, marked as answer]
<p>Hi again <a class="mention" href="/nickpan-fioneer">@nickpan-fioneer</a> :wave:t2: Closing the loop on this. Here is what I found out:</p>
<ol>
<li>If you have an enterprise trial account that you set up with sales support, you should contact your sales rep for assistance.</li>
<li>I found this on the GitHub Blog: <a href="https://github.blog/changelog/2021-11-10-github-enterprise-cloud-self-service-organization-upgrade-to-an-enterprise-account/">GitHub Enterprise Cloud self-service organization upgrade to an Enterprise Account</a> - but it does not mention trial accounts.</li>
<li>The GitHub public roadmap: <a href="https://github.com/github/roadmap">readme</a> + <a href="https://github.com/orgs/github/projects/4247">project board</a> has self-serve enterprise account features for the GHES trial planned, but there is no release date that can be shared publicly. I’ve shared your post with the engineering team in case they have more info to share.</li>
</ol>
<p>Thanks again for reaching out about this! I hope that’s helpful.</p>
